packages/shared/lib/db/migrations/20240125102526_remove_decoupled_provider_config_key.cjs

```diff
@@ -0,0 +1,22 @@
+const CONNECTIONS_TABLE = '_nango_connections';
+const CONFIGS_TABLE = '_nango_configs';
+
+exports.up = async function(knex) {
+    await knex.schema.withSchema('nango').alterTable(CONNECTIONS_TABLE, function (table) {
+        table.integer('nango_config_id').unsigned().references('id').inTable(`nango._nango_configs`).index();
```

Comment:
I would favor `config_id`. We have a lot of prefix everywhere, especially in the database, and I don't think it provides much value

Reply:
Updated

Reply:
did you pushed? 🤔

Reply:
Yeah, https://github.com/NangoHQ/nango/blob/master/packages/shared/lib/db/migrations/20240125102526_remove_decoupled_provider_config_key.cjs#L6